File: src/main/java/com/nickzim/DTOs/Attachments/Attachment.java
package com.nickzim.DTOs.Attachments;

import lombok.Data;

@Data
public class Attachment {

    private String type;

    private AttachmentObject photo;

    private AttachmentObject video;

    private AttachmentObject audio;

    private AttachmentObject doc;

    private AttachmentObject wall;

    private AttachmentObject sticker;

    private AttachmentObject link;
}

File: src/main/kotlin/com/nickzim/domain/attachments/Attachment.kt
package com.nickzim.domain.attachments

data class Attachment(
        val type: String?,
        val photo: AttachmentObject?,
        val video: AttachmentObject?,
        val audio: AttachmentObject?,
        val doc: AttachmentObject?,
        val sticker: AttachmentObject?,
        val link: AttachmentObject?
)
